karma/test-directives.coffee

```diff
@@ -110,5 +115,60 @@ describe('directives:', () ->
       expect(item2.querySelector('.pod-item-value').textContent)
         .toContain('Corge')
     )
+
+    it('should draw the pod actions', ->
+      $rootScope.podData.actions = [{
+        case_id: 21,
+        type: 'foo',
+        name: 'Foo',
+        payload: {bar: 'baz'}
+      }, {
+        case_id: 21,
+        type: 'quux',
+        name: 'Quux',
+        payload: {corge: 'grault'}
+      }]
+
+      el = $compile('<div cp-pod></div>')($rootScope)[0]
+      $rootScope.$digest()
+
+      action1 = el.querySelectorAll('.pod-action')[0]
+      action2 = el.querySelectorAll('.pod-action')[1]
+
+      expect(action1.textContent).toContain('Foo')
+      expect(action2.textContent).toContain('Quux')
+    )
+
+    it('should call trigger() when an action button is clicked', ->
+      $rootScope.podData.actions = [{
+        type: 'foo',
+        name: 'Foo',
+        case_id: 21,
+        payload: {a: 'b'}
```

Comment:
I'm not sure that we can send `case_id` when you read the pod data, since we have the same issue of not knowing which case the pod is for.

At the moment, each action in the list of actions returned has a `label` and `title` field (at least that's how it's documented). But that can easily change. I'll probably change it now to `type` (from `label`), and leave out the `title` field, and add the `payload` field.